docs/modules/ROOT/pages/references/component-library.adoc

```diff
@@ -63,6 +65,8 @@ When using this function, the caller is responsible to ensure that an `OperatorG
 
 The result of this function can be used in the same way as resources created by `kube.libjsonnet`.
 
+By default the components sets the nodeAffinity to infra nodes, this can be overwritten. See https://docs.openshift.com/container-platform/latest/nodes/scheduling/nodes-scheduler-node-affinity.html#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Official Documentation]
```

Comment:
```suggestion
By default the components sets the nodeAffinity to infra nodes, this can be overwritten.
See https://docs.openshift.com/container-platform/latest/nodes/scheduling/nodes-scheduler-node-affinity.html#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Official Documentation]
```